app/containers/Blog.jsx

```diff
@@ -1,43 +1,70 @@
-import React, { Component } from 'react';
-import { connect } from 'react-redux';
+import React from 'react';
+import { gql, graphql } from 'react-apollo';
+import { BLOG_SLUG, POSTS_PER_PAGE } from '../config/app';
 import { Head } from '../components/Common/Head';
 import { Title } from '../components/Content/Title';
-import { RenderContent } from '../components/Content/RenderContent';
 import { Loading } from '../components/Content/Loading';
 import { FourOhFour } from '../components/Content/FourOhFour';
 import { PostList } from '../components/Posts/PostList.jsx';
-import { fetchMorePosts } from '../actions/fetchMorePosts';
+import { postFragment } from '../utils/fragments';
 
-class Blog extends Component {
-  render() {
-    const { page, blog, settings, loading, params, fetchMorePosts, starwardUpdating } = this.props;
-    if (loading) return <Loading />;
-    if (!page || !blog) return <FourOhFour />;
-    return (
-      <main className="content" role="main">
-        <Head {...page.seo} defaultTitle={`${page.title} - ${settings.name}`} />
-        <Title title={page.title} />
-        <RenderContent content={page.content} />
-        <PostList
-          posts={blog}
-          currentPage={params.page}
-          starwardUpdating={starwardUpdating}
-          fetchMorePosts={fetchMorePosts}
-         />
-      </main>
-    );
-  }
-}
+const Blog = (props) => {
+  const { blog, settings, loading, categories, pagination } = props;
+  const { page, totalPages, totalItems } = pagination;
+  const pageTitle = 'blog';
+
+  if (loading) return <Loading />;
+  if (!blog) return <FourOhFour />;
+  return (
+    <main className="content" role="main">
+      <Head {...page.seo} defaultTitle={`${pageTitle} - ${settings.name}`} />
+      <Title title={pageTitle} />
```

Comment:
What happens when the blog is linked to a page with content that needs displaying above the post list?

Reply:
Not too sure what your asking? In the past this file was like this too, I see it more as an example usage of the data calls and how to use the data. I've always replaced everything inside of here.